### PR TITLE
Search

### PR DESCRIPTION
**Pull request by 02danieljonas**

### CODE REVIEW

File: flashcards-ui/src/components/FlashcardComponents/StudyPageContent/StudyPageContent.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        if (e.visibility == true) {
+            return e;
+        }
+    });
+
+    if (filteredFlashcard.length < 2) {
+        console.error(
+            `I don't know how but less than two flashcards are inside of filteredFlashcard, all flashcards will be used`
+        );
+        // console.log(filteredFlashcard)
```
Comment (YilikaLoufoua) [inline flashcards-ui/src/components/FlashcardComponents/StudyPageContent/StudyPageContent.jsx:18]: RECOMMENDED: Use Javascript Console Utils (VSCode Extension) to add and remove console logs easily.

File: flashcards-ui/src/components/MySetsPageComponents/StudyPage/StudyPage.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    } = useFlashcardContext();
+
+    const { isLoading, isLoggedIn } = useAuthContext();
+
+    const { setId } = useParams();
+    const navigate = useNavigate();
+
+    const { mySets } = useFlashcardContext();
+    const chosenSet = mySets.find((e) => e.id == setId);
+    // console.log(mySets);
```
Comment (YilikaLoufoua) [inline flashcards-ui/src/components/MySetsPageComponents/StudyPage/StudyPage.jsx:22]: Remove console log.

**Review by YilikaLoufoua — APPROVED** [review]
LGTM

File: flashcards-ui/src/components/FlashcardComponents/StudyPageContent/StudyPageContent.jsx (hunk 1 above)
Comment (02danieljonas) [reply flashcards-ui/src/components/FlashcardComponents/StudyPageContent/StudyPageContent.jsx:18]: That's just a placeholder for an alert system, should I still remove it?

**Pull request merged**